test/ng-currency/ng-currency-settings.provider.spec.js

```diff
@@ -18,7 +18,9 @@ describe('provider(ngCurrencySettings)', () => {
         hardCap: false,
         min: undefined,
         max: undefined,
-        currencySymbol: undefined
+        currencySymbol: undefined,
+        autoFill: false,
+        highlightOnFocus: true
```

Comment:
Rather then reusing tests for other bindings can you add a separate group for testing `highlightOnFocus` and `autoFill`.